jparest/src/test/java/uk/gov/homeoffice/digital/sas/jparest/controller/ResourceApiControllerTest.java

```diff
@@ -651,24 +658,78 @@ void batchUpdate_resourcesExist_persistsChanges() throws JsonProcessingException
         assertThat(checkResource2).isEqualTo(dummyTwo);
     }
 
-    @ParameterizedTest(name="{0}")
-    @MethodSource("invalidPayloads")
+    @Test
     @Transactional
-    void batchUpdate_invalidPayload_jsonExceptionThrown(String payload) {
-        var controller = getResourceApiController(DummyEntityA.class);
-        assertThatExceptionOfType(JsonProcessingException.class).isThrownBy(() -> controller.batchUpdate(TENANT_ID, payload));
+    void patch_oneResourceDoesntExist_resourceNotFoundExceptionThrown()
+        throws JsonProcessingException {
+
+        String createPayload = PayloadCreator.createPayload(Map.of(DESCRIPTION_FIELD_NAME, "Dummy Entity C",
+            INDEX_FIELD_NAME, 1, TENANT_ID_FIELD_NAME, TENANT_ID));
+
+        //create new resource
+        var controller = getResourceApiController(DummyEntityC.class);
+        var apiResponse = controller.create(TENANT_ID, createPayload);
+        assertThat(apiResponse.getItems()).hasSize(1);
+        var createdResource = apiResponse.getItems().get(0);
+
+        //get the newly created resource
+        var getResponse = controller.get(TENANT_ID, createdResource.getId());
+        var getResource = getResponse.getItems().get(0);
+        assertThat(getResource.getDescription()).isEqualTo("Dummy Entity C");
+
+        //create update payload
+        var updatedResourceOne = new DummyEntityC();
+        updatedResourceOne.setDescription("Updated Dummy Entity C One");
+        updatedResourceOne.setIndex(2L);
+        updatedResourceOne.setId(createdResource.getId());
+
+        var notCreatedResource = new DummyEntityC();
+        notCreatedResource.setId(NON_EXISTENT_ID);
+        notCreatedResource.setDescription("Updated Dummy Entity C Two");
+        notCreatedResource.setIndex(3L);
+
+        Object operationOne =
+            new PatchOperation<>("replace", "/" + createdResource.getId().toString(), updatedResourceOne);
```

Comment:
nit: noticed theres quite a few occurrences of this construction. 
Might be a bit neater and maintainable to have a static test util method somewhere to create a 'replace' based `PatchOperation` for us that just takes the `UUID` and value.